spec/PushWorker.spec.js

```diff
@@ -276,7 +408,6 @@ describe('PushWorker', () => {
           'failedPerType.ios': { __op: 'Increment', amount: 1 },
           [`sentPerUTCOffset.${UTCOffset}`]: { __op: 'Increment', amount: 1 },
           [`failedPerUTCOffset.${UTCOffset}`]: { __op: 'Increment', amount: 1 },
-          count: { __op: 'Increment', amount: -1 }
```

Comment:
If we're  not using `count` on _PushStatus we should probably strip it from the schema as well, unless there's some particular need for it still.